bluebrain/repo-bluebrain/packages/py-neurodamus/package.py

```diff
@@ -13,6 +13,7 @@ class PyNeurodamus(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/neurodamus-py.git"
 
     version('develop', branch='main', submodules=True)
+    version('2.12.9',  branch='magkanar/clear_model', submodules=True)
```

Comment:
No branches for fixed versions, please. Use a commit if you can't do a tag.

Reply:
Yes, this is just temporary to test everything together. I will replace this when the PR is ready for review. Should I also clear the PR build directory if I replace this with the tag and keep the same version?

Reply:
Yes, please. In fact, for every proper rebuild you'd have to clear the PR build directory (while testing)

Reply:
I've now updated the new py-neurodamus version with the proper tag. Thanks for your help!